octobot_trading/exchanges/parser/funding_rate_parser.py

```diff
@@ -0,0 +1,151 @@
+from octobot_trading.enums import ExchangeConstantsFundingColumns as FundingCols
+from octobot_trading.exchanges.parser.util import Parser
+import octobot_trading.exchanges.parser.util as parser_util
+from octobot_trading import constants as trading_constants
+from octobot_commons import constants as commons_constants
+
+
+class FundingRateParser(Parser):
+    """
+    overwrite FundingRateParser class methods if necessary
+    always/only include bulletproof custom code into the parser to improve generic support
+
+        parser usage:   parser = FundingRateParser(exchange)
+                        funding_rate = parser.parse_funding_rate(raw_funding_rate)
+                        funding_rates = parser.parse_funding_rate_list(raw_funding_rates)
+    """
+    
+    FUNDING_TIME_UPDATE_PERIOD = 8 * commons_constants.HOURS_TO_SECONDS
+
+    def __init__(self, exchange):
+        super().__init__(exchange=exchange)
+        self.PARSER_TITLE = "funding rate"
```

Comment:
Why should we use automatically the class name without "Parser" to create the title?

Reply:
its used for the error reporting, i need the name with and without parser, so error messages make sense